plugin/src/main/java/com/iridium/iridiumskyblock/upgrades/UpgradeData.java

```diff
@@ -0,0 +1,21 @@
+package com.iridium.iridiumskyblock.upgrades;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.iridium.iridiumskyblock.utils.Placeholder;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import java.util.Collections;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class UpgradeData {
```

Comment:
Why UpgradeData? It's literally an Upgrade so it should also have that name

Reply:
Because having two classes called upgrade is gonna be confusing af.

Reply:
What class are you referring too? Upgrades is a config and doesn't have the same name which is also true for the IslandUpgrade class which is a Java Bean and contains database logic.